localization/ndt_scan_matcher/README.md

```diff
@@ -34,6 +34,8 @@ One optional function is regularization. Please see the regularization chapter i
 | `points_aligned`                  | `sensor_msgs::msg::PointCloud2`                 | [debug topic] pointcloud aligned by scan matching                                                                                        |
 | `points_aligned_no_ground`        | `sensor_msgs::msg::PointCloud2`                 | [debug topic] de-grounded pointcloud aligned by scan matching                                                                            |
 | `initial_pose_with_covariance`    | `geometry_msgs::msg::PoseWithCovarianceStamped` | [debug topic] initial pose used in scan matching                                                                                         |
+| `multi_ndt_pose`                  | `geometry_msgs::msg::PoseArray`                 | [debug topic] estimated poses from multiple initial poses in real-time covariance estimation                                                                                                           |
+| `multi_initial_pose`              | `geometry_msgs::msg::PoseArray`                 | [debug topic] initial poses for real-time covariance estimation                                                                                                           |
```

Comment:
please delete unnecessary spaces
https://github.com/autowarefoundation/autoware.universe/pull/5454/commits/3744ee3525df05e16678c067af86bcd21e88c552

Reply:
@YamatoAndo 
I follow the Markdown table style lile in other sections.
Should I remove unnecessary spaces?

Reply:
yes, please remove spaces so that the tables align.

![Screenshot from 2023-11-02 13-42-06](https://github.com/autowarefoundation/autoware.universe/assets/13819566/a6cbb964-b174-4a19-98e7-10235f138133)

Reply:
Sorry, I was mistaken. I will fix them now.